### DISCUSSION TITLE
Link resolution in sent emails incorrect

### DISCUSSION THREAD

**Discussion by lazerusrm** [rustdesk/rustdesk-server-pro#457] [General]
Hello, im hosting my server at [RustdeskServer].[mydomain].com behind a nginx proxy and use nginx streaming for the additional ports. Everything works, following the install to set my site config correctly.

I initially installed the server and used the DNS option and set it to [RustdeskServer].[mydomain].com

When email invitations are sent. they are sent with the internal ip address, not the domain incorrectly. currently the format is

[http://[InternalIP]:21114/static/index.html#](http://[InternalIP]:21114/static/index.html#/user/verify?token=[Token]

I need it formatted http://[RustdeskServer].[mydomain].com/  (We are using port 80 and have SSL wildcard certs setup and working.)

The server works and everything, and i can manually send them an updated url, but its a nuisance. 

How can i set this configuration, i cant seem to find any config files, or anywhere to set these parameters.

Thanks!

**Comment by rustdesk** [comment]
@fufesou

**Comment by fufesou** [comment]
I'll fix it.

@lazerusrm For now, a fast solution is to modify the sqlite db directly.

```sql
update "team" set info = json_set(info,'$.smtp.base_url','YOUR-DOMAIN/') where name = 'Default';
```

**Comment by lazerusrm** [comment]
Thanks, worked. I wont include commands since, probably nobody should do this :)

**Reply by fufesou** [reply]
Yes, agree
